qemu/tests/block_resize.py

```diff
@@ -51,6 +53,24 @@ def verify_disk_size(session, os_type, disk):
                          current_size, block_size)
             return True
 
+    def get_md5_of_file(filename, create_flag=False):
+        """
+        Get the md5 value of filename
+        :param filename: Name of file to get md5 value.
+        :param create_flag: This file will be created if it is True.
+        """
+        if create_flag:
+            dd_cmd = params.get("dd_cmd")
+            if os_type == 'windows':
+                dd_cmd = utils_misc.set_winutils_letter(session, dd_cmd)
+
+            guest_cmd = dd_cmd % filename
+            session.cmd(guest_cmd)
+
+        guest_cmd = md5_cmd % filename
+        md5_val = session.cmd_output(guest_cmd).split()[0]
+        return md5_val
+
```

Comment:
I thought the function `get_md5_of_file`  should just get the md5 value. 
Better to follow `SRP(Single Responsibility Principle)`.